### PR TITLE
German enum values

### PR DESCRIPTION
**Pull request by dcsaszar**

### CODE REVIEW

File: src/Data/Quote/QuoteValuesConfig.ts [hunk 1]
```diff
@@ -81,8 +81,85 @@ provideDataValues(Quote, {
...
+    PSA_SAP_MAI_QUO_ORD_CRE: 'Angebot gewonnen',
+    PSA_SAP_MAI_QUO_ORD_EPH_VH: 'Verhandlungsphase',
+    PSA_SAP_MAI_QUO_ORD_EPH: 'Entscheidungsphase',
+    PSA_SAP_MAI_QUO_ORD_PRP_ZUR: 'Angebot zurückgestellt',
+  },
+  status: {
+    PSA_PRO_QUO_AS6: 'Angebot OK, Projektabschluss in 3-6 Monaten',
+    PSA_PRO_QUO_CMV: 'Angebot vom Kunden zurückgestellt',
+    PSA_PRO_QUO_COK: 'Angebot gewonnen/Kundenzusage',
+    PSA_PRO_QUO_CPE: 'Angebotsabgabe erfolgt, pending',
```
Comment (apepper) [inline src/Data/Quote/QuoteValuesConfig.ts:97]: This looks like a strange german translation.

File: src/Data/Ticket/TicketValuesConfig.ts [hunk 2]
```diff
@@ -37,7 +37,46 @@ provideDataValues(Ticket, {
...
+    PSA_SVC_CPL_FLW: 'nachbereitet',
+    PSA_SVC_CPL_ORD: 'Auftrag angelegt',
+    PSA_SVC_SUP_CPL_ASQ: 'erfasst',
+    PSA_SVC_SUP_CPL_BEA: 'in Bearbeitung durch Lieferanten',
+    PSA_SVC_SUP_CPL_ERL: 'erledigt',
+    PSA_SVC_SUP_CPL_RUE: 'rückgemeldet durch Lieferanten',
+    PSA_SVC_SUP_CPL_SEN: 'an Lieferanten gemeldet',
+    PSA_SVC_TRB_ACQ: 'erfasst',
+    PSA_SVC_TRB_CLS: 'beseitigt',
+    PSA_SVC_TRB_DCS: 'Abstellmaßnahme festgelegt',
```
Comment (apepper) [inline src/Data/Ticket/TicketValuesConfig.ts:66]: This status reads strange.

**Review by apepper — COMMENTED** [review]
FEEDBACK ([c83321f](https://github.com/Scrivito/scrivito-portal-app/pull/374/commits/c83321f3bb785dc5ca1ee5cccc4398c9ae5de998))

Code wise really nice 🚀! I only found some strange german locales.

File: src/Data/Quote/QuoteValuesConfig.ts (hunk 1 above)
Comment (dcsaszar) [reply src/Data/Quote/QuoteValuesConfig.ts:97]: According to Ronald these are the localizers from the standard. I don't see any potential todo here.

File: src/Data/Ticket/TicketValuesConfig.ts (hunk 2 above)
Comment (dcsaszar) [reply src/Data/Ticket/TicketValuesConfig.ts:66]: According to Ronald these are the localizers from the standard. I don't see any potential todo here.

**Pull request merged**